submissions/yanyshpolska/Hooli-style Popup/css/style.css

```diff
@@ -0,0 +1,213 @@
+/* base styles */
```

Comment:
please remove comments, they are not needed, code should be readable by its nature :D

Reply:
Agreed. My bad. I'll delete them.